app/client/components/management/exportManager.jsx

```diff
@@ -157,7 +157,9 @@ class ExportManager extends React.Component {
           </div>
 
           {this.state.exportType === "spreadsheet" ? transferToggle : null}
-
+          {this.props.export.message != undefined
```

Comment:
`message` is defined as `null | string` in `app/client/reducers/export/reducers.ts`. Shouldn't this be `this.props.export.message !== null`